### ISSUE TITLE
Wrong time zone

### ISSUE THREAD

**Issue by jakuub** [vacuumlabs/fine-tracker#93]
I logged time from 07:00 to 16:00 and in my tempo, it is shown from 06:00 to 15:00. 

![image](https://user-images.githubusercontent.com/1863153/50774605-24995880-1294-11e9-947c-d29903024e98.png)

It is possible, that I have something wrong setted up in tempo, but it should not show different hours, because it can cause problems when fixing something in tempo...

**Comment by misogl** [comment]
I have the same issue. My timezone is set to "Europe/Bratislava" in my Atlassian profile and the timezone on my computer is the same (`new Date().getTimezoneOffset() === -60`).

**Comment by jakuub** [comment]
I found the but. It is in Temp: 
Martin Stepanek, 2nd interview

Our timezone will correctly convert to UTC and store without timezone. But Tempo interprets time in database "relative to current timezone", so finetracker will interpret `1.1.2019 12:00 GMT` during saving to JIRA as `1.1.2019 11:00 UTC`, it is stored in JIRA database as `1.1.2019 11:00` (without timezone - standard for storing all in UTC). 
Tempo then take this data and interpret it as "relative to current timezone", so `1.1.2019 11:00 GMT`. This is known bug in Tempo:
https://jira.atlassian.com/browse/JRASERVER-35503 (it is closed, but not fixed)

It is a question, if we should be compatible with Tempo or with Jira...

**Comment by jozefhales** [comment]
Any progress on this issue? Or any suggested workaround at least?

**Comment by jakuub** [comment]
No, no progress on this issue. As we worked and used finetracker, it stopped bothering us. It is only a "cosmetic" issue in timelogs if you are logging time from slovakia and don't work across months - and then it will create 1-2 difference. 

I don't say we dropped this and don't want to resolve it somehow, but it didn't create so many problems at the end and it is not directly obvious, what is the correct solution for this problem. 

But we could open this with guys, who do reports and more high level staff with logs and discuss what approache we should use. 

But I think, that from users perspective, tempo-compatible solution will be to correct option. But I'm not sure.
